zds/utils/templatetags/interventions.py

```diff
@@ -126,7 +126,11 @@ def get_github_issue_url(topic):
     if not topic.github_issue:
         return None
     else:
-        return "{}/{}".format(settings.ZDS_APP["site"]["repository"]["bugtracker"], topic.github_issue)
+        if topic.github_repository_name:
+            repository_name = topic.github_repository_name
+        else:
+            repository_name = "zds-site"
+        return "{}/{}".format(get_repository(repository_name)["bugtracker"], topic.github_issue)
```

Comment:
Suggestion pour plus de lisibilité
```suggestion
        if topic.github_repository_name:
            repository_name = topic.github_repository_name
        else:
            repository_name = "zds-site"
        bugtracker = get_repository(repository_name)["bugtracker"]
        return f"{bugtracker}/{topic.github_issue}"
```